server/graphql/v2/interface/Account.js

```diff
@@ -568,6 +569,39 @@ const accountFieldsDefinition = () => ({
       };
     },
   },
+  childrenAccounts: {
+    type: AccountCollection,
+    args: {
+      limit: { type: GraphQLInt, defaultValue: 100 },
+      offset: { type: GraphQLInt, defaultValue: 0 },
```

Comment:
These ones too, since they have a defaultValue. It's good practice to enforce a `limit`